src/Smalot/PdfParser/Font.php

```diff
@@ -110,11 +110,17 @@ public function translateChar(string $char, bool $use_default = true)
         return $use_default ? self::MISSING : $fallbackDecoded;
     }
 
+    private static $uchrCache = [];
+    
     public static function uchr(int $code): string
     {
+        $code = (int) $code;
+		if (isset(self::$uchrCache[$code])) {
+			return self::$uchrCache[$code];
+		}
```

Comment:
```suggestion
        if (isset(self::$uchrCache[$code])) {
		    return self::$uchrCache[$code];
        }
```

Reply:
`$code = (int) $code;`

is not required because function sends you an `integer`.